packages/core/src/components/Button/Button.tsx

```diff
@@ -206,7 +206,6 @@ const Button: VibeComponent<ButtonProps, unknown> & {
         getStyle(styles, camelCase("kind-" + kind)),
         getStyle(styles, camelCase("color-" + calculatedColor)),
         {
-          [styles.loading]: loading,
```

Comment:
@YossiSaadi unused?

Reply:
this PR is dead, sorry 🥲 
and yes, it was removed as part of the Table New Sort behavior (#1935)